app/uk/gov/hmrc/eusubsidycompliancefrontend/models/json/eis/package.scala

```diff
@@ -123,9 +123,9 @@ package object eis {
   // provides response from EIS retrieve subsidies call
   implicit val eisRetrieveUndertakingSubsidiesResponse: Writes[UndertakingSubsidies] =
     new Writes[UndertakingSubsidies] {
-      // TODO delete this if we can get the case of subsidyUsageTransactionID aligned in SCP06 & 09
+      // TODO delete this if we can get the case of subsidyUsageTransactionId aligned in SCP06 & 09
       implicit val nonHmrcSubsidyWrites: Writes[NonHmrcSubsidy] = (
```

Comment:
I think we don't need to define write in this manner since the Id is sorted ? I may be wrong

Reply:
Oh good point. Just because it may introduce more bugs, do you mind if we raise a ticket to "rationalise explicit readers/writers"? There's probably room to clean up but I want to just be safe with this fix :)

Reply:
sure 👍